test/runtime/Policies.go

```diff
@@ -1087,20 +1086,36 @@ var _ = Describe("RuntimePolicies", func() {
 	It("Tests Egress To World", func() {
 		googleDNS := "8.8.8.8"
 		googleHTTP := "google.com"
-		checkEgressToWorld := func() {
-			By("Testing egress access to the world")
+		numberOfTries := 5
+		maxNumberOffFailures := 1
 
-			res := vm.ContainerExec(helpers.App1, helpers.Ping(googleDNS))
-			ExpectWithOffset(2, res).Should(helpers.CMDSuccess(),
-				"not able to ping %q", googleDNS)
+		checkEgressToWorld := func() {
 
-			res = vm.ContainerExec(helpers.App1, helpers.Ping(helpers.App2))
+			res := vm.ContainerExec(helpers.App1, helpers.Ping(helpers.App2))
 			ExpectWithOffset(2, res).ShouldNot(helpers.CMDSuccess(),
 				"unexpectedly able to ping %q", helpers.App2)
 
-			res = vm.ContainerExec(helpers.App1, helpers.CurlFail("-4 http://%s", googleHTTP))
-			ExpectWithOffset(2, res).Should(helpers.CMDSuccess(),
-				"not able to curl %s", googleHTTP)
+			By("Testing egress access to the world")
+			pingFailures := 0
+			curlFailures := 0
+
+			for i := 0; i < numberOfTries; i++ {
+				res := vm.ContainerExec(helpers.App1, helpers.Ping(googleDNS))
+				if !res.WasSuccessful() {
```

Comment:
Break out of the loop if ping and curl are successful. That way we won't have to connect multiple times unless we have to.

Reply:
Done!